### ISSUE TITLE
Allow for non-interactive installation

### ISSUE THREAD

**Issue by DoodlesEpic** [nextdns/windows#16]
It would be nice to have the installation succeed without the need to put a NextDNS ID into the installer and have to install the OpenVPN driver. Or maybe this prompt should appear only after installation if the user chooses to. This would be nice because the way NextDNS Windows Client currently installs makes it impossible to add it to the winget repos.

Related PRs in the winget repos:
https://github.com/microsoft/winget-pkgs/pull/60592
https://github.com/microsoft/winget-pkgs/pull/38207

**Comment by DoodlesEpic** [comment]
![image](https://user-images.githubusercontent.com/37254797/168401556-f756ee1f-a577-4717-8c82-8c5d3d0f3d13.png)
Having to install the device driver this way makes so it we can't add it to winget. Not sure if there's any possible solution to this coming from either NextDNS or Winget but I decided to create the issue to track progress on this as it would be nice to have this on Winget.

**Comment by rs** [comment]
Already fixed by a854440b5eabcf5b1e84d97bc307b594d91583e9. Please wait for version 3.0.0 release (soon).

**Comment by DoodlesEpic** [comment]
> Already fixed by [a854440](https://github.com/nextdns/windows/commit/a854440b5eabcf5b1e84d97bc307b594d91583e9). Please wait for version 3.0.0 release (soon).

Lightning fast response. Thanks so much! Will be on the lookout for 3.0

**Comment by rs** [comment]
You can try https://github.com/nextdns/windows/releases/tag/v3.0.0. Run it with the /PROFILE=abcdef argument to force a configuration during install. Doing so will prevent the systray UI from changing this value.
